storage/gcp/bucket/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0"
```

Comment:
Optionals require 1.3

Reply:
Do I update the version to 1.3 or do I have to remove the optionals inside the variables ?